textbook/12/chapter11-code-glossary.md

```diff
@@ -0,0 +1,36 @@
+## np.mean() 
+Calculates the mean of elements in an array.
+
+```python
+a = [10,20,30,40,50]
+np.mean([a])   # 30
+```
+
+## np.std() 
+Calculates the standard deviation of elements in an array.
+
+```python
+a = [10,20,30,40,50]
+np.std([a])   #14.14
+```
+
+## .sample()
```

Comment:
We should decide how we want to handle this. I've been doing things like 'df.sample()' because I think starting with the . might confused my glossary builder. But I also think starting with the . makes the distinction between methods and functions clearer to students.